.github/workflows/main.yml

```diff
@@ -18,7 +18,6 @@ jobs:
         run: |
           npm ci
           npm run lint
-          npm audit
```

Comment:
why removing this ?

Reply:
Yes I can keep et but I prefer to be alerted because sometimes vulnerabilities are in deep dependencies, so it's difficult to fix them before it's fixed in dependencies.

Reply:
ok I understand :)